src/twisted/internet/testing.py

```diff
@@ -414,8 +420,44 @@ def getDestination(self):
         return self._address
 
 
+@implementer(IHostResolution)
+@dataclass
+class ImmediateResolution:
+    name: str
+
+    def cancel(self) -> None:
+        """
+        Provided just for interface compliance; it should be impossible to
+        reach here, since it's resolved immediately.
+        """
+        raise Exception("already resolved")  # pragma: no cover
+
+
+@implementer(IHostnameResolver)
+class ImmediateResolver:
```

Comment:
minor comment

just some suggestion for some documentation and a more descriptive name

I am not arguing that my suggestion is a better name.

```suggestion
class FixedLoopbackResolver:
   """
   A simple resolver that will always resolve to `127.0.0.1`
   """
```

Reply:
OK. I will add a docstring and consider the name, perhaps adding a third option :)